src/CoreBundle/private/library/classes/dbobjects/TCMSFontImage.class.php

```diff
@@ -9,6 +9,9 @@
  * file that was distributed with this source code.
  */
 
+/**
+ * @deprecated since 6.3.0 - unusable since decades.
```

Comment:
```suggestion
 * @deprecated since 6.3.0 - unusable for decades.
```

I think we should also mark related database entities as deprecated. I know of the table cms_font_image which we can add to the UPGRADE file, hide in the classic main menu and remove from the new main menu.

Reply:
added the table, changed the text